### ISSUE TITLE
member-access - no-check-interface

### ISSUE THREAD

**Issue by superdyzio** [palantir/tslint#3948]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.7.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
export class ButtonComponent implements OnInit {
  @Input() public color : string;

  constructor() { }

  public ngOnInit() {
  }
}
```

with `tslint.json` configuration:

```json
    "member-access": true,
```

#### Actual behavior
member-access rule will always require 'public' keyword in front of lifecycle hooks methods from interfaces.

#### Expected behavior
It would be a nice feature to be able to tell tslint to ignore access modifiers in front of methods from interfaces but check them in front of any other methods and fields.

**Comment by giladgray** [comment]
@superdyzio i do not see the issue here. there is a rule that enforces that all members must have access modifiers, and `public` is the default one. if you're implementing some library hook then it must necessarily be `public` for external code to invoke it.

please add more detail if there is truly a bug here, but the current behavior seems desirable to me.
